README.md

```diff
@@ -1,2 +1,37 @@
-# metaframer
-Metaframer is a tool to generate frames for images from their metadata to be displayed alongside images in videos or other scenarios
+# Metaframer
+
+Welcome to Metaframer, the ultimate tool for automatically integrating metadata into stylish frames for your images. Use Metaframer to enhance the display of images in videos, digital galleries, presentations, and much more. Our easy-to-use features allow for dynamic, customizable, and visually appealing frame generation.
+
+## Features
+
+- **Automated Frame Creation:** Generates frames with metadata from images automatically.
+- **Customizable Designs:** Multiple frame designs to match your project's style.
+- **Dynamic Metadata Integration:** Automatically displays metadata such as date, location, camera settings.
+- **Flexible Customization:** Adjustable fonts, colors, and opacity.
+- **Broad Compatibility:** Works with a wide array of image formats and video editing software.
+- **Real-Time Preview:** Edit and see changes in real-time.
+- **Batch Processing:** Apply consistent frames to multiple images at once.
```

Comment:
```suggestion
- **Batch Processing:** Create consistent frames to multiple images at once.
```

We are not applying or editing the original images. `metaframer` only creates the frame for the images.